api/graphql/mutations/topic.js

```diff
@@ -1,4 +1,34 @@
-export function subscribe (userId, topicId, communityId, isSubscribing) {
-  return TagFollow.subscribe(topicId, userId, communityId, isSubscribing)
-  .then(() => ({success: true}))
+import { sanitize } from 'hylo-utils/text'
+
+export async function topicMutationPermissionCheck (userId, communityId) {
```

Comment:
just a note that some mutations use `userId` here and others use an `authZ` object which is
```
{
  userId,
  isAdmin
}
```
Not saying you need to switch for this PR. But at some point we should standardize that.